_data/toc.yaml

```diff
@@ -22,43 +22,49 @@ guides:
   section:
   - path: /engine/installation/
     title: Install Docker
-  - sectiontitle: Docker Edge
+  - sectiontitle: Docker EE
```

Comment:
I like the split! can we reverse the order so that we have `Docker CE` first? That's what people should use to get started with Docker, and also represents a bigger chunk of our installed base.